### PR TITLE
www.tecnoandroid.it

### PR DESCRIPTION
**Pull request by jellizaveta**
https://github.com/AdguardTeam/AdguardFilters/issues/120200

<details><summary>Screenshot:</summary>

Снимок экрана 2022-06-01 в 21 35 09

</details><br/>

### CODE REVIEW

File: EnglishFilter/sections/general_extensions.txt [hunk 1]
```diff
@@ -286,6 +286,9 @@ kat.*,kickass.*,kickass2.*,kickasstorrents.*,kat2.*,kattracker.*,thekat.*,thekic
 go.linkfly.io#%#//scriptlet("prevent-window-open")
 ! https://github.com/AdguardTeam/AdguardFilters/issues/120146
 go.bitcoinly.in#%#//scriptlet("prevent-window-open")
+! https://github.com/AdguardTeam/AdguardFilters/issues/120200
+tecnoandroid.it#%#//scriptlet("set-constant", "td_ad_background_click_link", "undefined")
+tecnoandroid.it#$#body.td-background-link { cursor: auto !important; }
```
Comment (Alex-302) [inline EnglishFilter/sections/general_extensions.txt:291]: Italian - to foreign.txt
the same for 
```
||tecnoandroid.it/wp-content/uploads/2022/04/vivo
||assets.evolutionadv.it/tecnoandroid_it/tecnoandroid_it.new.js
```

**Review by Alex-302 — CHANGES_REQUESTED** [review]

File: EnglishFilter/sections/general_extensions.txt (hunk 1 above)
Comment (zloyden) [reply EnglishFilter/sections/general_extensions.txt:291]: Yep, this is an Italian website

**Review by zloyden — CHANGES_REQUESTED** [review]

File: EnglishFilter/sections/specific.txt [hunk 2]
```diff
@@ -310,6 +310,8 @@ rushbitcoin.com,buxcoin.io,goldenclix.com##[class][style^="display:inline-block;
 mintscan.io##div[class^="Ad_container_"]
 ||mintscan.io/_next/static/image/assets/banner/
 ||apkmody.io/ads*.js
+||tecnoandroid.it/wp-content/uploads/2022/04/vivo
```
Comment (zloyden) [inline EnglishFilter/sections/specific.txt:313]: `2022/04/` is not necessary

File: EnglishFilter/sections/specific.txt [hunk 3]
```diff
@@ -310,6 +310,8 @@ rushbitcoin.com,buxcoin.io,goldenclix.com##[class][style^="display:inline-block;
...
+||assets.evolutionadv.it/tecnoandroid_it/tecnoandroid_it.new.js
```
Comment (zloyden) [inline EnglishFilter/sections/specific.txt:314]: Where do you need this?
Also move these rules to the `Italian` section

**Review by zloyden — CHANGES_REQUESTED** [review]

**Review by zloyden — APPROVED** [review]

**Pull request merged**